lucille-core/src/main/java/com/kmwllc/lucille/core/JsonDocument.java

```diff
@@ -477,6 +490,69 @@ public JsonNode getJson(String name) {
     return data.get(name);
   }
 
+  @Override
+  public Date getDate(String name) {
+    if (!data.has(name)) {
+      return null;
+    }
+
+    JsonNode node = getSingleNode(name);
+
+    long dateLong = node.asLong();
```

Comment:
Will this return an error if node is null? If so, we can just remove the dateLong variable and move node.asLong() directly into the Date constructor.

I don't think node should ever be null, since we first check if it exists on `data` but since we're checking node.isNull() in the return statement I guess it can be null?

Reply:
changed implementation to convert Date/Timestamp to Instant, and then delegating to the setters/getters of instant type methods